example/bwmf/bwmf/convert_pb.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/taskgraph/taskgraph/example/bwmf"
+	pb "github.com/taskgraph/taskgraph/example/bwmf/proto"
+	"github.com/taskgraph/taskgraph/filesystem"
+)
+
+func ConvertMatrixTxtToPb(confBytes []byte, path string) {
```

Comment:
Better separate it into two functions? One for convert and one for saving to fs.